### ISSUE TITLE
Calls: Emit audible status tones to inform caller about call progress

### ISSUE THREAD

**Issue by mwild1** [snikket-im/snikket-ios#191]
**Describe your feature proposal**
Currently when the app is establishing a call, no "ringing" tone is played, and in the case of failure, no error tones are played. The only way to get this information is to watch the screen.
